packages/core/src/formatters/utils.ts

```
@@ -16,6 +16,15 @@ export function formatBytes(
     value = Math.round(value);
   }
   return value + ' ' + sizes[i];
+  return value + ' ' + sizes[i];
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)